### PR TITLE
[Public/Global - Microsoft Azure Attestation Service] Adding August 2022 public Microsoft Attestation API version

### PR DESCRIPTION
**Pull request by olkroshk**
# Data Plane API - Pull Request

## API Info: The Basics
Most of the information about your service should be captured in the issue that serves as your [*engagement record*](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/271/Azure-REST-API-Stewardship?anchor=rest-api-stewardship-process).

* Link to engagement record issue:

Is this review for (select one):

- [ ] a private preview
- [ ] a public preview
- [x] GA release

### Change Scope
<sup>This section will help us focus on the specific parts of your API that are new or have been modified. <br/>Please share a link to the design document for the new APIs, a link to the previous Open API document (swagger) if applicable, and the root paths that have been updated. </sup>

SevSNP VM attestation type goes public and with the PR is exposed to the GA API of the Microsoft Azure Attestation service.

## ❔Got questions? Need additional info?? We are here to help!

<details>
  <summary> Contact us!</summary>

The [Azure API Review Board](https://aka.ms/azapi) is dedicated to helping you create amazing APIs. You can read about our mission and learn more about our process on our [wiki](https://aka.ms/azapi).
* 💬 [Teams Channel](https://teams.microsoft.com/l/channel/19%3a3ebb18fded0e47938f998e196a52952f%40thread.tacv2/General?groupId=1a10b50c-e870-4fe0-8483-bf5542a8d2d8&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47)
* 💌 [email](mailto://azureapirbcore@microsoft.com)

</details>

<details>
  <summary>Click here for links to tools, specs, guidelines & other good stuff</summary>

### Tooling
 * [Open API validation tools](https://aka.ms/swaggertools) were run on this PR. Go here to see [how to fix errors](https://aka.ms/ci-fix)
 * [Spectral Linting](https://aka.ms/style)
 * [Open API Hub](https://aka.ms/openapiportal)

### Guidelines & Specifications
 * [Azure REST API Guidelines](https://aka.ms/guidelines)
 * [OpenAPI Style Guidelines](https://aka.ms/style)
 * [Azure Breaking Change Policy](https://aka.ms/AzBreakingChangesPolicy)

### Helpful Links
 * [Azure DevTools Wiki](https://aka.ms/azapi)

</details>

### CODE REVIEW

File: specification/attestation/data-plane/Microsoft.Attestation/stable/2022-08-01/examples/AttestSgxEnclave.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
```
Comment (yinqchen) [inline specification/attestation/data-plane/Microsoft.Attestation/stable/2022-08-01/examples/AttestSgxEnclave.json:1]: Why this attest sevsnp example is different from the example in this PR?https://dev.azure.com/msazure/One/_git/security-Attestation/pullrequest/6780314?_a=files&path=/src/AttestationServices/RestApi/ReadOnlyGeneratedSwaggerFiles/attestation/data-plane/Microsoft.Attestation/stable/2022-08-01/examples/AttestSevSnpVm.json

Comment (kroshkina-ms) [reply specification/attestation/data-plane/Microsoft.Attestation/stable/2022-08-01/examples/AttestSgxEnclave.json:1]: I believe you are looking at the attest SGX enclave example..

**Comment by LarryOsterman** [comment]
It appears your branch is out of sync with main, there are a bunch of files here that don't appear to be related to your pull request.

**Comment by kroshkina-ms** [comment]
Closing this PR in favor of a new one https://github.com/Azure/azure-rest-api-specs/pull/20890 as there was an issue with the forks sync.

**Pull request closed without merging**